src/pxl_scripts/px/differential_flamegraph/vis.json

```diff
@@ -0,0 +1,66 @@
+{
+    "variables": [
+        {
+            "name": "start_time",
+            "type": "PX_STRING",
+            "description": "The relative start time of the window. Current time is assumed to be now",
+            "defaultValue": "-5m"
+        },
+        {
+            "name": "namespace",
+            "type": "PX_NAMESPACE",
+            "description": "The namespace to filter on."
+        },
+        {
+            "name": "pod",
+            "type": "PX_POD",
+            "description": "The pod that will have its flamegraph analyzed compared to the baseline_pod"
+        },
+        {
+            "name": "baseline_pod",
+            "type": "PX_POD",
+            "description": "The pod to serve as the baseline. The resulting flamegraph will show the difference from this pod's profile."
+        }
+    ],
+    "globalFuncs": [],
+    "widgets": [
+        {
+            "name": "Flamegraph",
+            "position": {
+                "x": 0,
+                "y": 0,
+                "w": 12,
+                "h": 6
+            },
+            "func": {
+                "name": "differential_flamegraph",
+                "args": [
+                    {
+                        "name": "start_time",
+                        "variable": "start_time"
+                    },
+                    {
+                        "name": "namespace",
+                        "variable": "namespace"
+                    },
+                    {
+                        "name": "pod",
+                        "variable": "pod"
+                    },
+                    {
+                        "name": "baseline_pod",
+                        "variable": "baseline_pod"
+                    }
+                ]
+            },
+            "displaySpec": {
+                "@type": "types.px.dev/px.vispb.StackTraceFlameGraph",
+                "stacktraceColumn": "stack_trace",
+                "countColumn": "count",
+                "percentageColumn": "percent",
+                "podColumn": "pod",
```

Comment:
a54f66098bc25560770fb1575f91621900a44083 added the `podColumn` column. From testing the original change (everything before a54f66098bc25560770fb1575f91621900a44083), I would very infrequently receive crashes on [this line](https://github.com/pixie-io/pixie/blob/4778ba1f1f28f0e7019e33747dfc016d4eaaae6c/src/ui/src/containers/live/convert-to-vega-spec/flamegraph.ts#L1019).

I unfortunately don't have the stack trace handy, but the only thing I can attribute it to is that it's possible for the `stacktraceColumn` field to result in an undefined or empty value that causes the crash. My suspicion is this doesn't happen for the `px/perf_flamegraph` script because it forces the scopedStackTrace variable to contain semicolons and `(k8s)` ([source](https://github.com/pixie-io/pixie/blob/4778ba1f1f28f0e7019e33747dfc016d4eaaae6c/src/ui/src/containers/live/convert-to-vega-spec/flamegraph.ts#L994-L1016)).